client/src/features/Event/Overview/components/NicknameModal.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from 'react';
+
+import { css } from '@emotion/react';
+import { useNavigate } from 'react-router-dom';
+
+import { useCreateProfile } from '@/api/mutations/useCreateProfile';
+import { Button } from '@/shared/components/Button';
+import { Flex } from '@/shared/components/Flex';
+import { Icon } from '@/shared/components/Icon';
+import { Input } from '@/shared/components/Input';
+import { Modal } from '@/shared/components/Modal/Modal';
+import { Text } from '@/shared/components/Text';
+
+type NicknameModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+export const NicknameModal = ({ isOpen, onClose }: NicknameModalProps) => {
+  //E.TODO 추후 organizationId 받아오기
+  const createProfileMutation = useCreateProfile(1);
+  const navigate = useNavigate();
+  const [nickname, setNickname] = useState('');
+
+  const handleNicknameSubmit = (nickname: string) => {
+    createProfileMutation.mutate(
+      { nickname },
+      {
+        onSuccess: () => {
+          alert('조직 참가가 완료되었습니다!');
+          navigate('/event');
+          onClose();
+        },
+        onError: () => {
+          alert('참가 중 오류가 발생했습니다. 다시 시도해 주세요.');
+        },
+      }
+    );
+  };
+
+  const handleSubmit = () => {
+    if (nickname.trim()) {
+      handleNicknameSubmit(nickname.trim());
+    }
+  };
+
+  const handleClose = () => {
+    setNickname('');
+    onClose();
+  };
+
+  return (
+    <Modal
+      isOpen={isOpen}
+      onClose={handleClose}
+      showCloseButton={false}
+      css={{ maxWidth: '360px' }}
```

Comment:
p1.
현재 글자가 max-width 만큼의 영역을 충분히 사용하지 않고, 내부 콘텐츠 크기만큼만 너비를 차지하고 있어 답답해 보이는 것 같아요. `width: 400px`으로 넓혀보는 건 어떠신가요??
```suggestion
      css={css`
        width: '400px'
      `}
```

Reply:
넵. 바꾸는 김에 css prop 사용으로도 같이 바꾸었습니다! 
4bcd50b7675e7739d8f6d778f7c8942114b5d73f